### ISSUE TITLE
Light mode color contrast may violate a11y

### ISSUE THREAD

**Issue by kxrt** [kxrt/pe#10]
![Screenshot 2022-11-11 at 5.08.40 PM.png](https://raw.githubusercontent.com/kxrt/pe/main/files/55288288-5b23-410d-a668-d4f7dabe51f0.png)

**Comment by nus-se-script** [comment]
[IMPORTANT!: Please do not edit or reply to this comment using the GitHub UI. You can respond to it using CATcher during the next phase of the PE]

# Team's Response
Web accessibility is undoubtedly an important aspect when it comes to designing applications. Regrettably, I overlooked this entirely when I was picking a colour scheme for the light theme, and should have known better.

Despite the validity of this issue, I am categorizing this as NotInScope due to accessibility not being an explicitly-mentioned deliverable for this project.

# Items for the Tester to Verify

## :question: Issue response

Team chose [`response.NotInScope`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your reason]

-------------------
